db/data/20200716074943_seed_accesses.rb

```diff
@@ -0,0 +1,11 @@
+class SeedAccesses < ActiveRecord::Migration[5.2]
+  def up
+    admin = EmailAuthentication.find_by_email("admin@simple.org").user
+    admin.accesses.admin.create!(resource: FacilityGroup.first)
```

Comment:
Is this just a starting point for testing? If so, could we just do this programmatically and avoid a data migration? I'd like to use data migrations for when we port everyone's permissions over, for example.